example_files/visualization/input_visualization_feature.html

```diff
@@ -483,13 +484,17 @@ <h2>Annual End Uses</h2>
                 y: endUseValue
               }]
             });
+
           });
-          $scope.annualEndUseChartOptions.chart['yDomain'] = [0, _.round(1.1 * annualTotalYMax)];
-        
+          if (annualTotalYMax > total_value_feature + 20000000) {
```

Comment:
The difference between the sum of all end uses for a commercial and residential building was ~ 20,000,000 kWh.